### PR TITLE
Test fixed

### LINKED ISSUE #5: Test breaking locally, travis passed
@barberboy 
Referencing #4 

Running `node fizzbuzz.js`, output looks correct. Though travis test passed, running test on desktop breaks with:

> assert.js:89
>   throw new assert.AssertionError({
>   ^
> ' on line 1.ror: Expected '1' to be '1
>     at C:\Users\James\Documents\GitHub\fizzbuzz\test\index.js:21:20
>     at FSReqWrap.readFileAfterClose [as oncomplete](fs.js:380:3)

### PR DESCRIPTION
**Pull request by keawade**
Was missing a `\r`. Not sure why it broke since last year. Resolves #5.

### CODE REVIEW

**Comment by keawade** [comment]
Oh no... Now it breaks on Travis. :disappointed:

I suspect it might be a Windows / Unix difference. I'll test that now.

**Comment by keawade** [comment]
The original `node test` fails on Windows in both Powershell and the Windows Subsystem for Linux but works fine on Ubuntu.

I've got an idea for how to work around the different line breaks. Will commit soon, hopefully.

**Comment by keawade** [comment]
Okay, the current test fails because I don't have a solution posted to this branch. But it no longer fails on due to incorrect line endings. I've tested on Windows and Ubuntu and they both work just fine. I think this can be merged in. @barberboy

**Comment by keawade** [comment]
Test confirmed to work on Travis. See #10.

**Comment by barberboy** [comment]
Thanks again for looking into this, Keith!

File: test/index.js [hunk 1]
```diff
@@ -14,7 +15,7 @@ exec('node .', function(err, stdout, stderr) {
             throw err;
         }
 
-        var expected = exp.split('\n');
+        var expected = platform() == 'win32' ? exp.split('\r\n') : exp.split('\n');
```
Comment (barberboy) [inline test/index.js]: We might be able to use [`os.EOL`](https://nodejs.org/api/os.html#os_os_eol) here instead to simplify.

``` js
var os = require('os')
//...
var expected = exp.split(os.EOS);
```

**Pull request merged**